### DISCUSSION TITLE
[urql] support the new `@populate` exchange

### DISCUSSION THREAD

**Discussion by nandorojo** [dotansimha/graphql-code-generator#9163] [Ideas]
<img width="863" alt="Screenshot 2023-03-13 at 3 52 00 PM" src="https://user-images.githubusercontent.com/13172299/224817078-91a7038c-4185-4707-9f02-6320f659d315.png">

URQL now supports a `@populate` directive. It would be nice for this to not break with the `client-preset` and to (somehow) work with the codegen. Docs: https://formidable.com/open-source/urql/docs/advanced/auto-populate-mutations/

**Comment by ardatan** [comment]
PRs are welcome!
